packages/admin-panel/src/pages/resources/DashboardItemsPage.js

```diff
@@ -42,6 +45,12 @@ const IMPORT_CONFIG = {
   },
 };
 
+const renderNewDashboardVizButton = () => (
+  <Link to="/viz-builder/dashboard-item/new">
```

Comment:
There seems to be a minor css issue here where the text shows up with an underline underneath:
![image](https://user-images.githubusercontent.com/59544282/142957403-f2b32562-4d7b-406e-a061-94aa49c9b9b9.png)

I think wrapping the `Link` component as a styled component with no `text-decoration` can fix this, something like:
```
const StyledLink = styled(Link)`
  text-decoration: none;
  &:focus,
  &:hover,
  &:visited,
  &:link,
  &:active {
    text-decoration: none;
  }
`;
```

Reply:
Updated